wrappers/cl

```diff
@@ -18,8 +18,10 @@
 
 # /showIncludes
 unixify_path='/^Note: including file: /{ s/z:([\\/])/\1/i; s,\\,/,g; }'
+# /E
+unixify_line='/^[[:blank:]]*#line[[:blank:]]/{ s/z:([\\/])/\1/i; s,\\\\,/,g; }'
```

Comment:
Should we tolerate blanks between `#` and `line` too? I guess MSVC doesn't produce that, but just as `#   define FOO` is somewhat common as indentation, I guess `#  line` also could be considered accepted? (Likewise, I guess we could accept `[[:blank:]]+` after `line`?)

Reply:
I actually tested this. The preprocessor compresses all blank characters in `#    line    5    "a.c"`, and becomes `#line 5 "a.c"`. But blank characters before `#` are preserved.

Reply:
Ok, good to know. I meant more from the point of view of being flexible in accepting what MSVC might output, i.e. it could change to not normalize those lines in that way, in a future update. I.e., not only matching exactly what we get today, but matching anything that semantically means the same thing.

Reply:
ok, fair enough.

Reply:
Oh, right, that's true, as we're not covering the file name itself with the RE, only selecting which lines to apply it on.